packages/vue_frontend_rpc/local-dev/backend-mock.js

```diff
@@ -44,24 +49,13 @@ class BackendMock {
       const remoteAddress = req.socket.remoteAddress;
       console.log(`new ws connection from: ${remoteAddress}`);
       rpc = new RpcExtensionWebSockets(ws);
-
-      rpc.registerMethod({
-        name: "onFrontendReady",
-        func: async () => {
-          console.log(`Frontend is ready!`);
-          return actualOpts.onFrontendReady();
-        },
-        thisArg: null,
-      });
-
-      rpc.registerMethod({
-        name: "save",
-        func: async (newEditorFileObject) => {
-          console.log(`Saving updated contents!`);
-          return actualOpts.save(newEditorFileObject);
-        },
-        thisArg: null,
-      });
+      for (const i in backendFuncNames) {
```

Comment:
we can iterate over the values of the array directly (for of / .forEach)
no need to deal with the `i` index variable.